### ISSUE TITLE
[HOLD #26793][Wave 6: Tags] CRITICAL - Backend updates to EditMoneyRequest to enable editing of tag and billable flag

### ISSUE THREAD

**Issue by amyevans** [Expensify/App#26792]
## Design doc section
* https://docs.google.com/document/d/1eXGKhdUP98f6RUVYpRzxLi49El9Rizeqq-EYjrF24tU/edit#heading=h.98g6bdhi8f9i

## Implementation
- [x] https://github.com/Expensify/Web-Expensify/pull/38987
- [x] https://github.com/Expensify/Auth/pull/8784

**Comment by amyevans** [comment]
cc @mountiny were you interested in this one?

**Comment by mountiny** [comment]
@amyevans as mentioned in the doc, I think for the edit money request we first need to do the frontend changes (pass these values to the backend before we can create the command otherwise it will keep resetting the tag value as it assumes user added an empty tag).

Ie the hold should be the other way around

**Comment by amyevans** [comment]
Ah okay, sorry for misunderstanding before! But it makes sense now 😄 I'll update the titles.

**Comment by mountiny** [comment]
still on hold

**Comment by puneetlath** [comment]
@amyevans I believe everything is done here, correct?

**Comment by amyevans** [comment]
Ah yep, all set!

**Comment by puneetlath** [comment]
🎉
